spec/requests/portfolios_rbac_read_access_spec.rb

```diff
@@ -33,6 +35,35 @@
         get "#{api('1.0')}/portfolios/#{portfolio2.id}", :headers => default_headers
         expect(response).to have_http_status(:forbidden)
       end
+
+      context "via graphql" do
+        let(:graphql_query) { "{ portfolios(id: #{portfolio1.id}) { id name } }" }
+        it 'ok' do
+          post "#{api('1.0')}/graphql", :headers => default_headers, :params => graphql_body
+          expect(response).to have_http_status(:ok)
+          expect(json['data']['portfolios'][0]['name']).to eq(portfolio1.name)
+        end
+      end
+
+      context "via graphql" do
```

Comment:
This context is the same as in line 39. You may want to rephrase them to be more meaningful.